readme/testing.md

```diff
@@ -169,3 +169,10 @@ Project level, functional PHPUnit tests are included in `tests/phpunit`. Any PHP
 * [Test Driven Development: By Example (book)](http://www.amazon.com/dp/0321146530)
 * [xUnit Test Patterns: Refactoring Test Code (book for the really serious)](http://amazon.com/dp/0131495054)
 * [Unit testing: Why bother?](http://soundsoftware.ac.uk/unit-testing-why-bother/)
+
+### Configuration
+
+You can [customize the configuration values](extending-blt.md#modifying-blt-configuration) for the phpunit key. Each should containn a path (to a phpunit test or directory that  contains at least one test) and a config path (if custom configuration or boostrapping of Drupal is required):
```

Comment:
You can customize the `tests:phpunit` command by modifying the [customize the configuration values](extending-blt.md#modifying-blt-configuration) for the `phpunit` key. 

Each row under the `phpunit` key should contain a `path` and a `config` key. The `path` key indicates a directory containing PHPUnit tests. The optional `config` key can be used to specify a PHP configuration file.